docs/connecting-data/connecting-to-apis-and-external-data/peopledatalabs-api.md

```diff
@@ -2,6 +2,179 @@
 description: How to Connect DataDistillr to the People Data Labs API
 ---
 
-# People Data Labs API
+# Connecting to People Data Labs
 
-Details coming soon!
\ No newline at end of file
+## Creating a People Data Labs account
+Set up an account with [People Data Labs](https://www.peopledatalabs.com/signup).
+
+### Costs
+This API has three pricing plans:
+
+**Startup:** FREE TRIAL
+: $0.
+
+**Traction:** MONTHLY API PACKAGES
+: Tiered.
+
+**Scale:** ENTERPRISE
+: Custom. For enterprises building platforms with people data at scale. [Speak to a data consultant](https://www.peopledatalabs.com/talk-to-sales)
+
+For more information about People Data Labs pricing visit [https://www.peopledatalabs.com/pricing](https://www.peopledatalabs.com/pricing)
+
+### Rate Limits
+
+The default limit for free customers is 100/min. The default limit for paying customers is 1000/min.
+
+#### The API Dashboard
+
+All accounts are given access to a [dashboard](https://www.peopledatalabs.com/main), which will allows you to manage your API keys, view usage and test new endpoints (when they are available).
+
+## How to Connect DataDistillr to People Data Labs
+To set up a data source connection for People Data Labs, you will need to have:
+
+- A [unique name](#name) for your data source connection to be used in queries
+- An [API token](#api-key) generated through your People Data Labs account
+
+
+### Data Source Form
+To locate the People Data Labs form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.
+
+<figure markdown>
+  ![Data Source Wizard][image-9]{ width="100%" }
+</figure>
+
+On the API screen, select People Data Labs from the list of API forms.
+
+<figure markdown>
+  ![List of APIs][image-3]{ width="100%" }
+</figure>
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the People Data Labs API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+
+<figure markdown>
+  ![People Data Labs Form][image-5]{ width="100%" }
+</figure>
+
+
+### Name
+Enter any name that will help you recognize this data source within your query window.
+
+!!! info "Acceptable Characters Include"
+
+    - lowercase alphanumeric characters
+    - underscores
+
+
+### API Key
+The API key is generated within your account page. The following steps will navigate you to its location. Once created, copy the key and enter it in the People Data Labs form under 'API Key'.
+
+!!! example "Steps for getting the API key"
+
+    === "1. Home page"
+    
+        Click the **API Keys** button
+
+        ![Home page][image-1]
+
+    === "2. Account settings"
+
+        This is page contains the settings for your API token. You can modify the token's nickname and generate a new key in with **Roll This Key**.
```

Comment:
Wording here is a little awkward. Maybe modify the following: This page contains* and generate a new key with*